client/component/comment.js

```diff
@@ -250,11 +250,11 @@ function EditComment({comment, big=false, setComment, topLevel, onEditingDone, o
     </View>
 }
 
-function EditWidgets({widgets, comment, setComment, onCancel}) {
+function EditWidgets({widgets, comment, setComment, preselectedPerspectiveText, onCancel,}) {
```

Comment:
Can we make this more generic. E.g. pass all the params that the composer screen had, rather than just this special case one.

Reply:
done